server/sonar-server/src/main/java/org/sonar/server/issue/notification/IssueChangesEmailTemplate.java

```diff
@@ -100,9 +103,18 @@ private static void appendHeader(Notification notif, StringBuilder sb) {
     appendField(sb, "Message", null, notif.getFieldValue("message"));
   }
 
-  private void appendFooter(StringBuilder sb, Notification notification) {
+  private void appendFooter(StringBuilder sb, Notification notification){
     String issueKey = notification.getFieldValue("key");
-    sb.append("See it in SonarQube: ").append(settings.getServerBaseURL()).append("/issues?issues=").append(issueKey).append(NEW_LINE);
+    try {
+      sb.append("See it in SonarQube: ").append(settings.getServerBaseURL())
+        .append("/project/issues?id=").append(encode(notification.getFieldValue("projectKey"), "UTF-8"))
+        .append("&issues=").append(issueKey)
+        .append("&open=").append(issueKey)
+        .append(NEW_LINE);
+    }
+    catch (UnsupportedEncodingException e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this "catch" on the same line that the previous closing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)